src/mon/MgrStatMonitor.h

```diff
@@ -14,12 +14,14 @@ class MgrStatMonitor : public PaxosService {
   PGMapDigest digest;
   ServiceMap service_map;
   std::map<std::string,ProgressEvent> progress_events;
+  mempool::pgmap::map<uint64_t, PoolAvailability> pool_availability;
```

Comment:
please explain the use of mempool..

Reply:
Okay I see, I think mempool is unnecessary here, we can opt-in for just a regular "map". I think originally put that down because I saw a similar implementation elsewhere in the monitor code and so I thought if I were to do things "pool-related" it might give me a better optimization.

Reply:
Removed usage of mempool!